### PR TITLE
Add variable declaration node

### PR DESCRIPTION
**Pull request by nm723**
I fixed issue #13 , and also implemented reassignment (AssignmentExpression node).

### CODE REVIEW

File: src/utils/nodes/MemberExpression.ts [hunk 1]
```diff
@@ -40,13 +40,23 @@ export class MemberExpressionClass extends NodeClass {
...
-							nodeFactory({
-								node: arg,
-								context: this.context,
-								params: this.params,
-							}).serialize()
-						)
+						?.map((arg) => {
+							if (arg.type === 'Identifier') {
+								const variableValue = this.context.variables[arg.name];
+								if (!variableValue) return '';
```
Comment (nm723) [inline src/utils/nodes/MemberExpression.ts:46]: Currently, when writing
```
let a;
console.log(a);
```
a blank line is output in the Console. What kind of behavior would be expected?

Comment (vault-developer) [reply src/utils/nodes/MemberExpression.ts:46]: Hey @nm723 thank you for the question.
We want to simulate browser behaviour, therefore `undefined` seems like an appropriate output in such case.

Comment (nm723) [reply src/utils/nodes/MemberExpression.ts]: Thank you for the advice. I changed the output to `undefined`.

**Comment by vault-developer** [comment]
Hey @nm723 thank you for the PR!

**Pull request merged**